uavcan/driver/__init__.py

```diff
@@ -28,9 +29,12 @@ def make_driver(device_name, **kwargs):
     """
     windows_com_port = device_name.replace('\\', '').replace('.', '').lower().startswith('com')
     unix_tty = device_name.startswith('/dev/')
+    pythoncan = device_name.startswith('pythoncan')
 
     if windows_com_port or unix_tty:
         return SLCAN(device_name, **kwargs)
+    elif pythoncan is not None:
```

Comment:
Have you tested this code? This condition will never be true because `pythoncan` is of type `bool`, hence can't be None.

Reply:
No, I did not test this code. I now tested it. The code is somewhat simpler, but works now. No option to set the bitrate via the `kwargs` parameter. In anyway, we set it in the can.ini file.